### ISSUE TITLE
Login doesn't work if username is used instead of email

### ISSUE THREAD

**Issue by hbh7** [PlexRipper/PlexRipper#194]
Hi,
I just tried setting this up and logging into my Plex account. Validation passes with no issue, but saving doesn't do anything at all, only showing the following in the logs with nothing afterwards:
```
plexripper  | 02:47:00 [Debug] [PlexApi.PlexSignInAsync] => Requesting PlexToken for account <account username>
plexripper  |
plexripper  | 02:47:00 [Debug] [PlexApiClient.SendRequestAsync] => Sending request to: https://plex.tv/api/v2/users/signin
plexripper  |
plexripper  | 02:47:01 [Debug] [RestSharpExtensions.GenerateResponseResult] => Created:
plexripper  |
plexripper  | 02:47:01 [Debug] [RestSharpExtensions.GenerateResponseResult] => StatusCode - 201:
plexripper  |
plexripper  | 02:47:01 [Information] [PlexApiService.PlexSignInAsync] => Successfully retrieved the PlexAccount data for user <account username> from the PlexApi
plexripper  |
plexripper  | 02:47:01 [Debug] [PlexAccountService.ValidatePlexAccountAsync] => The PlexAccount with displayName <account username> has been validated
plexripper  |
plexripper  | 02:47:01 [Debug] [PlexAccountService.CreatePlexAccountAsync] => Creating account with username <account username>
```

Browser response log for the request to `/api/PlexAccount`:
`{"isFailed":true,"isSuccess":false,"reasons":[{"message":"Fluent Validation Pipeline Failed.","metadata":{"StatusCode":400}}],"errors":[{"reasons":[{"reasons":[]}]}],"successes":[]}`

If I use my account email instead of the username, it works instantly, and was a relief after a little while of troubleshooting and searching for other people with this issue haha. Since I found no one else experiencing this, I thought I'd make an issue myself. Happy to provide any other details as needed. Thanks!

**Comment by JasonLandbridge** [comment]
Hey thanks for reporting this! I just tested it out and PlexRipper works with just an username and password. HOWEVER, *insert dramatic pause*, Plex has a hidden option which you only see if you click on your username in the Plex account settings:

![image](https://user-images.githubusercontent.com/15127381/225125846-60c409ac-c75c-4f70-8902-df1bb57b573f.png)

Yeah that is just bad UI design on their part, so that might have been the issue.

It could also be that I just released a new version and accidentally fixed this, but let's just blame Plex for this one...

Regardless, I will add this to the F.A.Q on the website!
